ui/src/components/MainContainer.js

```diff
@@ -63,6 +65,18 @@ const MainContainer = () => {
           permissions={["public"]}
           component={VisitUsPage}
         />
+        <PrivateRoute
+          exact
+          path="/complete-profile"
+          permissions={["unsigned", "signed", "alumni", "admin", "stewardship"]}
```

Comment:
not a comment, but more of a question. How should we handle permissions for this page? I'm assuming a person who has already completed this page shouldn't be able to access this page anymore?

Reply:
yea they shouldn't actually. so this is hard cos we need a flag or indicator that someone has already completed their profile. i have no way to verify that at the moment.